Adjusted magento versions link (#1196)
mercadopago/devsite-docs: guides/plugins/official/magento-two.en.md

> Important
>
> Before setting up Pix as a payment method, remember to:<br><br>

> > - [Download the latest version](https://marketplace.magento.com/mercadopago-core.html#product.info.details.release_notes) to update your Mercado Pago plugin.<br>

Reviewer: ```suggestion
> - [Verify the latest version](https://marketplace.magento.com/mercadopago-core.html#product.info.details.release_notes) and update your Mercado Pago plugin.<br>
```
